Geovalidation.md

```diff
@@ -0,0 +1,143 @@
+#Geovalidation API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [Request Structure](#request-structure)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+#Description
+
+Accepts structured or unstructured location information and tests the validity of the resulting resolved geographic coordinate based on the strategies requested and the location types accepted. This service supports the HTTP/GET method only.
+
+----------------------------
+#Request Structure
+
+Example URL: https://api.careerbuilder.com/core/geography/validation?locality=atlanta&country=US&validation_strategies=FREETEXT&accepted_location_types=locality
+
+At least one of (query, locality, postal_code, admin_area, country) must be provided in the query string, as well as a comma separated strings of valid accepted location types, and validation stratgies.
```

Comment:
"as well as a comma separated strings of valid accepted location types, and validation stratgies."

-->

"as well as a "validation_strategies" parameter and an "accepted_location_types" parameter, each a comma-separated list of one or more values."